extensions/wikia/Search/classes/Config.php

```diff
@@ -1075,40 +1233,50 @@ public function mustAddMatchedRecords() {
 	/**
 	 * Adds a filter query based on the optional key, or automatically incremented key
 	 * Note that if you provide a key that already exists, you are overwriting that filter query.
+	 *
 	 * @param  string $queryString
 	 * @param  string $key
-	 * @return \Wikia\Search\Config
+	 *
+	 * @return Config
 	 */
 	public function setFilterQuery( $queryString, $key = null ) {
 		$key = $key ?: sprintf( 'fq%d', ++self::$filterQueryIncrement );
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Extract this increment or decrement operator into a dedicated statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS881)